app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -109,6 +111,7 @@ class WireActivityViewModel @Inject constructor(
     private val observeNewClients: ObserveNewClientsUseCase,
     private val clearNewClientsForUser: ClearNewClientsForUserUseCase,
     private val currentScreenManager: CurrentScreenManager,
+    private val observeScreenshotCensoringConfigUseCase: ObserveScreenshotCensoringConfigUseCase,
```

Comment:
this will crash the app when opened if there is no user logged in
the usecase can be changed to never rely on user sessions

- observe accounts
- observe each account self delete settings
- if only one it enforced, then enforce it for the whole app
this will set if for all of the accounts even if only one have it enforced 

or 
- in wire activity observe current account
- for that account only observe the ObserveScreenshotCensoringConfigUseCase
this will make it per accounts


in both cases the in app switch is global regardless of the account

Reply:
Nice catch 👍  I did the second option